commcare_connect/program/forms.py

```diff
@@ -1,12 +1,12 @@
 from crispy_forms.helper import FormHelper
 from crispy_forms.layout import Button, Field, Layout, Row, Submit
-from crispy_tailwind.tailwind import CSSContainer
 from django import forms
 
 from commcare_connect.opportunity.forms import OpportunityInitForm
 from commcare_connect.organization.models import Organization
 from commcare_connect.program.models import ManagedOpportunity, Program, ProgramApplicationStatus
-from commcare_connect.utils.forms import DATE_INPUT, FORM_BASE_STYLE
+
+DATE_INPUT = forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"})
```

Comment:
Why move this out of a shared utils file that would allow other forms to use the same code?

Reply:
This is getting used only in the `program/forms.py` file. I will make another PR to abstract common form stuff out from all places into the utils once the all the form PRs are merged.